pom.xml

```diff
@@ -244,7 +242,7 @@
         <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
         <jts.version>1.14.0</jts.version>
         <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
-        <karaf.version>4.2.1</karaf.version>
+        <karaf.version>4.2.2</karaf.version>
```

Comment:
✏️[[KARAF-5851] Remove heading spaces in the cfg files](https://github.com/apache/karaf/commit/5057aa441eb2df2f676d669b5bf0a5f223064e9f#diff-59845b1c772d324b5d5834b2724c7350)
https://github.com/codice/ddf/blob/8c982450cba9d8b7c12bc7c348c9ce7145acdcf5/features/karaf/src/main/feature/feature.xml#L55-L85
```xml
        <config name="org.apache.aries.transaction">
################################################################################
#
#    Licensed to the Apache Software Foundation (ASF) under one or more
#    contributor license agreements.  See the NOTICE file distributed with
#    this work for additional information regarding copyright ownership.
#    The ASF licenses this file to You under the Apache License, Version 2.0
#    (the "License"); you may not use this file except in compliance with
#    the License.  You may obtain a copy of the License at
#
#       http://www.apache.org/licenses/LICENSE-2.0
#
#    Unless required by applicable law or agreed to in writing, software
#    distributed under the License is distributed on an "AS IS" BASIS,
#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
#    See the License for the specific language governing permissions and
#    limitations under the License.
#
################################################################################

aries.transaction.recoverable = true
aries.transaction.timeout = 600
aries.transaction.howl.logFileDir = ${karaf.data}/txlog
aries.transaction.howl.maxLogFiles = 2
aries.transaction.howl.maxBlocksPerFile = 512
aries.transaction.howl.bufferSize = 4
        </config>
```